addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -37,6 +37,13 @@ type antreaProxy struct {
 	ProxyLoadBalancerIPs bool     `yaml:"proxyLoadBalancerIPs,omitempty"`
 }
 
+type antreaFlowExporter struct {
+	Address       string `yaml:"address,omitempty"`
```

Comment:
I have a suggestion for the names:
- Address -> CollectorAddress
- ExportTimeout -> ActiveFlowTimeout
- IdleTimeout  -> IdleFlowTimeout